src/styles/Search.css

```diff
@@ -0,0 +1,27 @@
+.search-input {
+  width: 100%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  border: 1px solid #dfe1e5;
+  border-radius: 20px;
+  box-sizing: border-box;
+  outline: 0;
+}
+
+.search-input:hover {
+  border-color: rgba(223, 225, 229, 0);
+  box-shadow: 0 1px 6px 0 rgba(32, 33, 36, 0.28);
+}
+
+.clear-button {
+  position: absolute;
+  color: #65676a;
+  top: 1rem;
```

Comment:
Not any kind of blocker, just a suggestion: I notice a mix of measurement units in the CSS here (px & rem) I might suggest picking one and sticking with it for consistency sake (em & rem or px).